iroh-embed/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "iroh-embed"
+version = "0.1.1"
+edition = "2021"
+license = "Apache-2.0/MIT"
+description = "Embedable iroh library"
+rust-version = "1.63"
+
+[dependencies]
+anyhow = "1"
+iroh-api = { path = "../iroh-api" }
```

Comment:
Other workspace crates add a version to this dependency.  Why?

Reply:
so we can publish to crates.io, otherwise when this is published it can not resolve the path dep